### PR TITLE
Add man page for ocid

### PR DESCRIPTION
**Pull request by rhatdan**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -9,6 +9,7 @@ OCID_INSTANCE := ocid_dev
 SYSTEM_GOPATH := ${GOPATH}
 PREFIX ?= ${DESTDIR}/usr
 INSTALLDIR=${PREFIX}/bin
+GO_MD2MAN ?= /usr/bin/go-md2man
```
Comment (runcom) [inline Makefile]: worth adding `go-md2man` to `make install.tools` - feel free to skip this comment, I'll make a follow up maybe

**Comment by runcom** [comment]
Dan, you need to sign the commit

File: Makefile (hunk 1 above)
Comment (rhatdan) [inline Makefile]: Fine

File: Makefile [hunk 2]
```diff
@@ -61,10 +65,21 @@ localintegration: binaries
...
+docs/%.8: docs/%.8.md
+	$(GO_MD2MAN) -in $< -out $@.tmp && touch $@.tmp && mv $@.tmp $@
+
+docs: $(MANPAGES_MD:%.md=%)
+
+install: binaries docs
 	install -D -m 755 ocid ${INSTALLDIR}/ocid
 	install -D -m 755 ocic ${INSTALLDIR}/ocic
 	install -D -m 755 conmon/conmon ${INSTALLDIR}/conmon
+	install -D -m 644 $(basename $(MANPAGES_MD)) $(PREFIX)/share/man/man8
```
Comment (mrunalp) [inline Makefile]: Remove them on uninstall?

Comment (rhatdan) [inline Makefile]: I figured out a way to remove them on uninstall.  Just wanted to make sure if we add multiple man pages that it would work.

**Comment by rhatdan** [comment]
@mrunalp PTAL

**Comment by mrunalp** [comment]
LGTM. We can add to travis in follow up.

**Comment by runcom** [comment]
> LGTM. We can add to travis in follow up.

yup, taking care of it just now

**Pull request merged**